install/mysql/glpi-9.2-empty.sql

```diff
@@ -251,6 +251,27 @@ CREATE TABLE `glpi_savedsearches_users` (
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 
 
+### Dump table glpi_savedsearches_alerts
+
+DROP TABLE IF EXISTS `glpi_savedsearches_alerts`;
+CREATE TABLE `glpi_savedsearches_alerts` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `savedsearches_id` int(11) NOT NULL DEFAULT '0',
+  `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+  `is_active` tinyint(1) NOT NULL DEFAULT '0',
+  `operator` tinyint(1) NOT NULL,
```

Comment:
A tinyint, not an int :p Well, this is lighter than a varchar as far as I remember.

Making DB readable is not really something I care about generally; as far as we use constants in code.